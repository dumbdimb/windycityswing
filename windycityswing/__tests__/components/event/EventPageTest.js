import React from 'react';
import renderer from 'react-test-renderer';

import EventPage from '..\\..\\..\\src\\scripts\\components\\event\\EventPage';

it('renders correctly', () => {
    expect(renderer.create(<EventPage match={{params: {eventName: 'test-event-name'}}}/>))
});