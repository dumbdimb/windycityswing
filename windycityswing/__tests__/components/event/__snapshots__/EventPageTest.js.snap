// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="event-details"
>
  <span
    className="event-details__title"
  >
    Fizz
  </span>
  <span
    className="event-details__header"
  >
    LINKS
  </span>
  <a
    className="event-details__link event-details__link--facebook"
    href="https://www.facebook.com/Fizz-Swing-Dancing-73706742234/"
    target="_blank"
  >
    <span>
      Facebook
    </span>
    <svg
      className="event-details__link-icon"
      fill="transparent"
      stroke="white"
      viewBox="0 0 16 16"
    >
      <path
        d="M8 3 L1 3 L1 15 L13 15 L13 8"
      />
      <path
        d="M10 1 L15 1 L15 6"
      />
      <line
        x1="7"
        x2="15"
        y1="9"
        y2="1"
      />
    </svg>
  </a>
  <span
    className="event-details__header"
  >
    DESCRIPTION
  </span>
  <span
    className="event-details__description"
  >
    Occurs every Monday.
  </span>
  <span
    className="event-details__description"
  >
    All ages event.
  </span>
  <span
    className="event-details__header"
  >
    PRICE
  </span>
  <span
    className="event-details__price"
  >
    $7/person for general public, $5/person for students
  </span>
  <span
    className="event-details__header"
  >
    SCHEDULE
  </span>
  <table>
    <tbody>
      <tr>
        <td
          className="event-details__schedule__time"
        >
          8:00 PM
        </td>
        <td
          className="event-details__schedule__event"
        >
          Swing lesson
        </td>
      </tr>
      <tr>
        <td />
        <td
          className="event-details__schedule__duration event-details__schedule__duration-one"
        >
          1 hr
        </td>
      </tr>
      <tr>
        <td
          className="event-details__schedule__time"
        >
          9:00 PM
        </td>
        <td
          className="event-details__schedule__event"
        >
          Dancing
        </td>
      </tr>
      <tr>
        <td />
        <td
          className="event-details__schedule__duration event-details__schedule__duration-three"
        >
          3 hr
        </td>
      </tr>
      <tr>
        <td
          className="event-details__schedule__time"
        >
          Midnight
        </td>
        <td
          className="event-details__schedule__event"
        >
          End
        </td>
      </tr>
    </tbody>
  </table>
  <span
    className="event-details__header"
  >
    DRESS CODE
  </span>
  <span
    className="event-details__dress-code"
  >
    Casual
  </span>
</div>
`;
